lib/tasks/launch_promo.rake

```diff
@@ -0,0 +1,28 @@
+require "#{Rails.root}/app/helpers/promos_helper"
+include PromosHelper
```

Comment:
I have a feeling that `include` in the top level namespace might be a bad idea

i think moving the include to the task body might be okay?